diff --git a/node_modules/@buger/probe/src/utils.js b/node_modules/@buger/probe/src/utils.js
index 2d25bb9..116d7f0 100644
--- a/node_modules/@buger/probe/src/utils.js
+++ b/node_modules/@buger/probe/src/utils.js
@@ -25,7 +25,11 @@ let probeBinaryPath = '';
 export async function getBinaryPath(options = {}) {
 	const { forceDownload = false, version } = options;
 
-	// Return cached path if available and not forcing download
+  if (options.path) {
+    return options.path;
+  }
+
+  // Return cached path if available and not forcing download
 	if (probeBinaryPath && !forceDownload && fs.existsSync(probeBinaryPath)) {
 		return probeBinaryPath;
 	}
